client/src/components/form/Form.tsx

```diff
@@ -0,0 +1,222 @@
+                                    >
+                                        Scale
+                                        <span className="text-red-500">*</span>
+                                    </label>
+                                    <TextInput
+                                        id="scale"
+                                        defaultValue="10000"
+                                        value={scale.toLocaleString()}
+                                        onValueChange={(v) => {
+                                            if (v === "") {
+                                                setScale(0);
+                                                return;
+                                            }
+                                            const num = parseLocalizedNumber(v);
+                                            if (!Number.isNaN(num) && Number.isInteger(num) && num >= 0 && num <= 10_000_000_000_000) {
+                                                setScale(num);
+                                            }
+                                        }}
+                                        name="scale"
+                                        autoComplete="scale"
+                                        placeholder="10.000"
+                                        className="mt-2"
+                                        icon={() =>
+                                            <p
+                                                className="border-r h-full text-tremor-default text-end text-right tremor-TextInput-icon shrink-0 h-5 w-5 mx-2.5 absolute left-0 flex items-center text-tremor-content-subtle dark:text-dark-tremor-content-subtle"
+                                            >
+                                                1:
+                                            </p>
+                                        }
+                                        required
+                                    />
+                                </div>
+
+                                <div className="col-span-full sm:col-span-3">
+                                    <label
+                                        htmlFor="language"
+                                        className="text-tremor-default font-medium text-tremor-content-strong dark:text-dark-tremor-content-strong"
+                                    >
+                                        Language
+                                    </label>
+                                    <SearchSelect
+                                        id="language"
+                                        name="language"
+                                        className="mt-2"
+                                    >
+                                        {
+                                            locales.map((l) => {
+                                                return <SearchSelectItem value={l.code}>{l.name}</SearchSelectItem>
```

Comment:
add key={l.name} on SearchSelectItem. 
these three changes are related to the error I saw on console of the browser which was: "Each child in a list should have a unique "key" prop"